src/marqo/tensor_search/backend.py

```diff
@@ -108,7 +108,7 @@ def add_customer_field_properties(config: Config, index_name: str,
             }
         }
     }
-    existing_info = get_cache()[index_name]
+    existing_info = get_index_info(config=config, index_name=index_name)
```

Comment:
to what extent will this change slow the `add_documents` process?

Reply:
thanks - updated.